k8s/perftest/common/divorce/div-cms.yaml

```diff
@@ -27,29 +27,12 @@ spec:
       readinessPeriod: 15
       useInterpodAntiAffinity: true
       applicationInsightsInstrumentKey: "eddbcec4-6ebc-4507-9f22-38d53a92e761"
-      image: hmctspublic.azurecr.io/div/cms:prod-d2b80c6c
-      ingressHost: div-cms-perftest.service.core-compute-perftest.internal
+      image: hmctspublic.azurecr.io/div/cms:prod-33c59844
```

Comment:
To do: update this once https://github.com/hmcts/div-case-orchestration-service/pull/484 is merged